### DISCUSSION TITLE
Horizontal scroll bar for bookmark

### DISCUSSION THREAD

**Discussion by beitadoro** [sumatrapdfreader/sumatrapdf#2866] [Q&A]
Hi, is there a way to turn on horizontal scroll bar for bookmark?
It will be helpful in navigating a long pdf with many bookmarks.

**Comment by GitHubRulesOK** [comment]
generally should be no need, 
  [/soapbox mode on] 
  The bookmarks should be a brief unique marker short and sweet. (like turning down a small corner or paperclip) not writing war and piece in a margin.
  [/soapbox mode off]

You don't need to scroll simply hover should show the marker for either favorites or Table of Content/Outlines
![image](https://user-images.githubusercontent.com/12754913/184667125-b36ce894-3126-464a-a0a4-9fc40e8727cb.png)
